internal/pattern/DefaultHttpPattern.json

```diff
@@ -20,10 +20,6 @@
         "sub": "=$.conf.jwtSub"
       }
     },
-    {
```

Comment:
This call to the circuit breaker is needed. Why did you remove it?

Reply:
Fixed